src/com/lilithsthrone/game/inventory/clothing/AbstractClothingType.java

```diff
@@ -237,319 +235,190 @@ public AbstractClothingType(
+			.map(Element.applyToContent(Integer::valueOf)) //<enchantmentLimit>x</enchantmentLimit>, x being Integer		
+			.orElse(-1);// empty element or no value in element, assign default value;
+		
+		this.clothingSet = coreAttributes.getOptionalFirstOf("clothingSet")
+			.filter(filterEmptyElements)
+			.map(Element.applyToContent(ClothingSet::valueOf))
+			.orElse(null);
+		
+		this.pathName = clothingXMLFile.getParentFile().getAbsolutePath() + "/" 
+			+ coreAttributes.getMandatoryFirstOf("imageName").getTextContent();
 				
-				this.itemTags = defaultItemTags;
+		this.pathNameEquipped = coreAttributes.getOptionalFirstOf("imageEquippedName")
+			.filter(filterEmptyElements)
+			.map(Element::getTextContent)
+			.orElse(null);
+		
+		Function< Element, List<Colour> > getColoursFromElement = (colorsElement) -> { //Another helper function to get the colors depending on if it's a specified group or a list of individual colors
+		    if(colorsElement.getAttribute("values").isEmpty()){
+			return colorsElement.getAllOf("colour").stream()
+				.map(Element.applyToContent(Colour::valueOf))
+				.collect(Collectors.toList());
+		    }
+		    else{
+			return ColourListPresets.valueOf(colorsElement.getAttribute("values"))
+				.getPresetColourList();
+		    }
+		};
+		
+		List<Colour> importedPrimaryColours = getColoursFromElement
+			.apply(coreAttributes.getMandatoryFirstOf("primaryColours"));	
+		List<Colour> importedPrimaryColoursDye = getColoursFromElement
+			.apply(coreAttributes.getMandatoryFirstOf("primaryColoursDye"));		
+		
+		List<Colour> importedSecondaryColours = coreAttributes.getOptionalFirstOf("secondaryColours")
+			.map(getColoursFromElement::apply)
+			.orElse(new ArrayList<>()); // ArrayList::new doesn't work here						
+		List<Colour> importedSecondaryColoursDye = coreAttributes.getOptionalFirstOf("secondaryColoursDye")
+			.map(getColoursFromElement::apply)
+			.orElse(new ArrayList<>());
 				
-				finalSetUp();
-
-			} catch(Exception ex) {
-				System.err.println("ClothingType was unable to be loaded from file!");
-			}
-		}
+		List<Colour> importedTertiaryColours = coreAttributes.getOptionalFirstOf("tertiaryColours")
+			.map(getColoursFromElement::apply)
+			.orElse(new ArrayList<>());		
+		List<Colour> importedTertiaryColoursDye = coreAttributes.getOptionalFirstOf("tertiaryColoursDye")
+			.map(getColoursFromElement::apply)
+			.orElse(new ArrayList<>());
+		
+		setUpColours(
+			importedPrimaryColours,
+			importedPrimaryColoursDye,
+			importedSecondaryColours,
+			importedSecondaryColoursDye,
+			importedTertiaryColours,
+			importedTertiaryColoursDye
+		);
+		
+		finalSetUp();
+	    }
+	    catch(XMLLoadException ex){		//rethrow if same type
+		throw ex;
+	    }
+	    catch(Exception ex){		//Other exceptions: NumberFormatException, XMLMissingTagException
```

Comment:
multicatch if you intended to only catch those exceptions

I think you should - null pointers here SHOULD crash the game because that's something that needs fixing